sql/codegen.go

```diff
@@ -213,32 +207,15 @@ STEP = 1000
 NUM_BUCKETS=160000
 EMBEDDING_WIDTH=128
 
-{{if eq .Driver "mysql"}}
-db = connect(user="{{.User}}",
-            passwd="{{.Password}}",
-            {{if ne .Database ""}}database="{{.Database}}",{{end}}
-            host="{{.Host}}",
-            port={{.Port}})
-{{else if eq .Driver "sqlite3"}}
-db = connect({{.Database}})
-{{else if eq .Driver "hive"}}
-db = connect(user="{{.User}}",
-            password="{{.Password}}",
-            {{if ne .Database ""}}database="{{.Database}}",{{end}}
-            host="{{.Host}}",
-            port={{.Port}})
+driver="{{.Driver}}"
+{{if ne .Database ""}}
+database="{{.Database}}"
 {{else}}
-raise ValueError("unrecognized database driver: {{.Driver}}")
+database=None
 {{end}}
 
-cursor = db.cursor()
-cursor.execute("""{{.StandardSelect}}""")
-{{if eq .Driver "hive"}}
-field_names = [i[0][i[0].find('.')+1:] for i in cursor.description]
-{{else}}
-field_names = [i[0] for i in cursor.description]
-{{end}}
-columns = list(map(list, zip(*cursor.fetchall())))
+conn = connect(driver, database, user="{{.User}}", password="{{.Password}}", host="{{.Host}}", port={{.Port}})
+field_names, columns = execute(driver, conn, """{{.StandardSelect}}""")
 
```

Comment:
How about in the future PRs, let us do something more:

- Move the following code snippet that constructs `feature_columns` and `feature_column_name` into a Python file, say, `naive_feature_dreivation.py`, and make it a function `derive_features`?

- Move the code snippet that constructs the classifier into `create_estimator.py` and make it a function `create_estimator`?

- Move the function `train_input_fn` and `eval_input_fn` and the logic that calls them to train and to evaluate the model into `train.py`?

Reply:
Yes, I will do this in the future PRs after both considering the code in `codegen.go` and `codegen_alps.go`.